molecule/opensearch/cleanup.yml

```diff
@@ -10,9 +10,9 @@
       {{ lookup('ansible.builtin.env', 'BRANCH_NAME') }}
     build: >-
       {{ lookup('ansible.builtin.env', 'BUILD_NUMBER') }}
-    aws_base_resource_name: "{{ ['molecule', it_platform, repo_branch, build] | join('_') }}"
+    ec2_resource_name: "{{ ['molecule', it_platform, repo_branch, build] | join('_') }}"
     domain_name: >-
-      {{ aws_base_resource_name | ansible.builtin.regex_replace('[^a-zA-Z0-9]','-') | lower | truncate(28, True, '') }}
+      {{ ['molecule', it_platform, build, repo_branch] | join('-') | lower | truncate(28, True, '') }}
```

Comment:
not many alternative ideas:

I've swapped build and repo branch vars so at least we avoid truncating the build number which is the most variable thing in the resource naming, so instead of:

```
molecule_opensearch_opsexp_1
```
we get:

```
molecule_opensearch_123456_o
```

Reply:
can't we deal we "transitions" problem manually to move from one naming model to another and replace the `molecule`prefix with a tag?